ci-operator/step-registry/openshift/e2e/aws/builds/techpreview/openshift-e2e-aws-builds-techpreview-workflow.yaml

```diff
@@ -0,0 +1,14 @@
+workflow:
+  as: openshift-e2e-aws-builds-techpreview
+  steps:
+    pre:
+    - chain: ipi-aws-pre-techpreview
+    test:
+    - ref: openshift-e2e-test
+    post:
+    - chain: gather-core-dump
+    - chain: ipi-aws-post
+    env:
+      TEST_SUITE: openshift/build
```

Comment:
not that I've done this before, but yes this is what I was suggesting @jkhelil 

now, over time, I suspect we'll create a new test suite in openshift/origin for build tests that we only run in the tech preview mode, but IMO at least that is outside the scope of your Jira.

when build csi volumes is completed, they'll need to make those updates and tweak this line

Reply:
alright, thanks @gabemontero

Reply:
> now, over time, I suspect we'll create a new test suite in openshift/origin for build tests that we only run in the tech preview mode, but IMO at least that is outside the scope of your Jira.

Alternatively, our tests can check if tech preview has been enabled, invoking appropriate skips if tech preview is or is not enabled.

Reply:
yep another valid option